os_builders/roles/vm_baseline/tasks/wazuh.yml

```diff
@@ -0,0 +1,135 @@
+- name: Install wazuh on Ubuntu
+  block:
+    - name: Add GPG Key for Wazuh
+      ansible.builtin.apt_key:
+        url: "https://packages.wazuh.com/key/GPG-KEY-WAZUH"
+        state: present
+
+    - name: Add Wazuh repo
+      ansible.builtin.apt_repository:
+        repo: "deb https://packages.wazuh.com/4.x/apt/ stable main"
+        state: present
+
+    - name: Install Wazuh agent
+      ansible.builtin.apt:
+        name: wazuh-agent=4.8.*
```

Comment:
Can we pin the major version of Wazuh and let me minor version float too? I'm assuming they use semver so 4.9 would be compatible with a 4.8 server?

Reply:
Unfortunately the agent is not forwards compatible:
```
Compatibility between the Wazuh agent and the Wazuh manager is guaranteed when the Wazuh manager version is later than or equal to that of the Wazuh agent.
```